breakpad-symbols/src/lib.rs

```diff
@@ -52,6 +52,13 @@ pub use crate::sym_file::{CfiRules, SymbolFile};
 
 mod sym_file;
 
+// Public re-exports for the purposes of the cfi_eval fuzzer.
+#[doc(hidden)]
```

Comment:
Assuming doc hidden is good enough to make this not public API. I can add a feature flag as well that the fuzzing crate sets.

Reply:
It might be good to hide this behind `#[cfg(feature = "fuzz")]`, yeah.